server/src/controllers/Events/resolver.ts

```diff
@@ -795,17 +772,32 @@ ${unsubscribeOptions}`,
     });
     await deleteEventReminders(id);
 
+    const chapterURL = `${process.env.CLIENT_LOCATION}/chapters/${event.chapter.id}`;
     const notCanceledRsvps = event.event_users;
-
+    const unsubScribeOptions = getUnsubscribeOptions({
+      chapterId: event.chapter_id,
+      eventId: event.id,
+      userId: ctx.user.id,
```

Comment:
wait because the one who activating this function is instance owner, does that mean if someone tries to unsubscribe, they will use instance owner id?

Reply:
If the instance owner makes the request, yes. Otherwise, no.